server/src/async/events.js

```diff
@@ -0,0 +1,8 @@
+const EVENTS = {
+  MEASURABLES_STATES: 'MEASURABLES_STATES',
```

Comment:
If this is an EVENT, can we change it to something like, ``RUN_MEASURABLE_STATE_TRANSITIONS``

Reply:
Event it is not a command.

Reply:
Even so. Even ``MEASURABLE_STATE_TRANSITIONS`` would be more specific.

Reply:
Thanks, ok.